martinMutuma/def-politico
# #164813252 Update BASE_URL endpoint

**What does this PR do?**


Updates the `BASE_URL` to the new API URL
**Description of Task to be completed?**



-   [x] Update Endpoint/api url to point to new api resource location

**How should this be manually tested?**

-   Clone the repo
-   Check out to this branch `ch/164813252-update-base-url`
-   Open the home page in your browser and see everything still works as expected.


**What are the relevant pivotal tracker stories?**



[#164813252](https://www.pivotaltracker.com/n/projects/2320126/stories/164813252)
